sculptor-version/src/main/kotlin/io/papermc/sculptor/version/SculptorVersion.kt

```diff
@@ -157,25 +171,27 @@ class SculptorVersion : Plugin<Project> {
             patchDir.set(target.layout.projectDirectory.dir("patches"))
         }
 
-        target.tasks.register("runServer", JavaExec::class) {
-            group = "mache"
-            description = "Runs the minecraft server"
-            doNotTrackState("Run server")
+        if (mache.minecraftJarType.getOrElse(MinecraftSide.SERVER) == MinecraftSide.SERVER) {
```

Comment:
this is always true, the mache extension isn't evaluated at this point in time yet, needs to be done in target.afterEvaluate further down in order to take effect